tests/acceptance_crontab_spec.lua

```diff
@@ -0,0 +1,216 @@
+        after_each(function()
+            -- Disable plugin and clear diagnostics
+            vim.cmd("CronExplainedDisable")
+            
+            -- Reset buffer to original
+            vim.api.nvim_set_current_buf(orig_buf)
+            pcall(vim.api.nvim_buf_delete, buf, { force = true })
+        end)
+        
+        it("Activates plugin for .crontab files", function()
+            -- Setup plugin with .crontab in file_patterns
+            cronex.setup({
+                file_patterns = { "*.crontab" },
+                explainer = {
+                    cmd = { "echo", "test-explanation" }
+                }
+            })
+            
+            -- Set current buffer and manually trigger the BufEnter autocmd
+            vim.api.nvim_set_current_buf(buf)
+            vim.cmd("doautocmd BufEnter")
+            
+            -- Wait for async operations to complete
+            wait()
+            
+            -- Get plugin namespace
+            local ns = vim.api.nvim_get_namespaces()["plugin-cronex.nvim"]
+            assert.is_not_nil(ns, "Plugin namespace should exist")
+            
+            -- Get diagnostics
+            local diags = vim.diagnostic.get(buf)
+            assert.is_not_nil(diags, "Diagnostics should exist")
+            assert.is_true(#diags > 0, "Should have diagnostics for cron line")
+            
+            -- Verify the diagnostic is on the correct line and has expected content
+            local cron_diag = diags[1]
+            assert.are.equal(0, cron_diag.lnum, "Diagnostic should be on the first line")
+            assert.truthy(string.match(cron_diag.message, "test%-explanation"), 
+                        "Diagnostic message should contain explainer output")
+            assert.are.equal(ns, cron_diag.namespace, "Diagnostic should be in the plugin namespace")
+            
+            -- Verify cleanup works
+            vim.cmd("CronExplainedDisable")
+            local after_diags = vim.diagnostic.get(buf)
+            assert.are.same({}, after_diags, "Diagnostics should be cleared after disabling")
+        end)
+        
+        it("Manually enables for .crontab file", function()
+            -- Setup plugin without .crontab in patterns
+            cronex.setup({
+                file_patterns = { "*.yml" }, -- Deliberately exclude crontab
+                explainer = {
+                    cmd = { "echo", "test-explanation" }
+                }
+            })
+            
+            -- Set buffer and check no diagnostics yet
+            vim.api.nvim_set_current_buf(buf)
+            local before_diags = vim.diagnostic.get(buf)
+            assert.are.same({}, before_diags, "Should have no diagnostics before enabling")
+            
+            -- Manually enable
+            vim.cmd("CronExplainedEnable")
+            wait()
+            
+            -- Check diagnostics after manual enable
+            local after_diags = vim.diagnostic.get(buf)
```

Comment:
same here, it'd be great to check for the cronex-specific diags (by passing namespace opt arg to vim.diagnostic.get